src/main/java/ca/corefacility/bioinformatics/irida/ria/web/ajax/projects/RemoteProjectInfo.java

```diff
@@ -0,0 +1,76 @@
+package ca.corefacility.bioinformatics.irida.ria.web.ajax.projects;
+
+import java.util.Date;
+
+import ca.corefacility.bioinformatics.irida.model.RemoteAPI;
+import ca.corefacility.bioinformatics.irida.model.project.ProjectSyncFrequency;
+import ca.corefacility.bioinformatics.irida.model.remote.RemoteStatus;
+import ca.corefacility.bioinformatics.irida.model.user.User;
+
```

Comment:
Comment

Reply:
Added in bb34052964426a9d5f05269592dbeb56afcf1e82